src/main/Config.h

```diff
@@ -116,14 +116,40 @@ class Config : public std::enable_shared_from_this<Config>
 
     typedef std::shared_ptr<Config> pointer;
 
+    // These test modes should be used in the following contexts:
+    // 1. TESTDB_DEFAULT / TESTDB_BUCKET_DB_VOLATILE: provides the most
+    //    comprehensive end-to-end test, but does not support arbitrary ledger
+    //    state writes via ltx root commits. All ledger state changes must occur
+    //    via applying valid TXs or manually adding entries to the BucketList.
+    //    BucketList state is not preserved over restarts. If this mode can be
+    //    used, it should be.
+    // 2. TESTDB_IN_MEMORY_NO_OFFERS: allows arbitrary ledger state writes via
```

Comment:
do we need both `TESTDB_IN_MEMORY_NO_OFFERS` and `TESTDB_IN_MEMORY_OFFERS`? seems like we can get away with just a single mode that stores everything, no?